data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -806,26 +802,11 @@ def build_academy_data(
         axis=1,
     )
 
-    academies["Status"] = academies.apply(
-        lambda df: mappings.map_academy_status(
-            pd.to_datetime(df["Date joined or opened if in period"], dayfirst=True),
-            pd.to_datetime(df["Date left or closed if in period"], dayfirst=True),
-            pd.to_datetime(df["Valid To"], dayfirst=True),
-            pd.to_datetime(df["OpenDate"]),
-            pd.to_datetime(df["CloseDate"]),
-            pd.to_datetime(accounts_return_period_start_date),
-            pd.to_datetime(academy_year_start_date),
-            pd.to_datetime(academy_year_end_date),
-        ),
-        axis=1,
-    )
-
+    # TODO: should factor into apportionment.
```

Comment:
Think this might be an artefact of a merge/rebase? This TODO was removed recently, IIRC.